lib/src/flagsmith_client.dart

```diff
@@ -88,6 +100,60 @@ class FlagsmithClient {
     return null;
   }
 
+  Future<void> _setupRealtimeUpdates(String realtimeUpdatesBaseUrl) async {
+    final sseUrl = '$realtimeUpdatesBaseUrl$apiKey/stream';
+
+    _sseStream = SSEClient.subscribeToSSE(
```

Comment:
Maybe just check that you don't have to close() the existing stream here if you're getting a new one. Probably not but it might be a memory leak without it.

Reply:
It throws an error if I call `SSEClient.unsubscribeFromSSE()` so I'll leave it as it is.